arrow-site/docs/docs/quickstart/README.md

```diff
@@ -7,28 +7,199 @@ permalink: /core/
 
-- [Patterns]({{ '/patterns/glossary/' | relative_url }}): tutorials and approaches to day-to-day challenges using FP
-- [Libraries]({{ '/quickstart/libraries/' | relative_url }}): all the libraries provided by Λrrow
-- [Effects]({{ '/effects/async/' | relative_url }}): interfacing with external systems
-- [Optics]({{ '/optics/iso/' | relative_url }}): inspecting and modifying data structures
+## [Fx]({{ '/fx/' | relative_url }})
+Arrow Fx is a full-featured, high-performance, asynchronous framework that brings functional operators to Kotlin's `suspend` functions.
+By leveraging the power of KotlinX Coroutines and the compiler support for CPS transformations, Arrow Fx results in optimal async programs with increased throughput and decreased allocations.
 
-## Curated external links
+## [Optics]({{ '/optics/' | relative_url }})
+Arrow Optics provides an automatic DSL that allows users to use `.` notation when accessing, composing, and transforming deeply nested immutable data structures.
+Optics also offers all the base types such as [Lens]({{ '/optics/lens/' | relative_url }}), [Prism]({{ '/optics/prism/' | relative_url }}), and others from which we can generalize accessing and traversing deep values in sealed and data classes models.
 
-- [Projects and Examples]({{ '/quickstart/projects/' | relative_url }})
-- [Media](https://media.arrow-kt.io)
+## [Meta]({{ '/meta/' | relative_url }})
+Arrow Meta is a general purpose library for meta-programming in Kotlin to build compiler plugins.
+Some type system features proposed by Arrow such as union types, product types, proof derivation, and others are built with Arrow Meta and serve as examples of what could be incorporated in the Kotlin compiler.
 
 ## Setup
 
-Take a look at [Setup]({{ '/quickstart/setup/' | relative_url }}) where you'll find the instructions to configure a project with `Gradle` or `Maven` to use `Arrow` library.
+### Next development version
+
+If you want to try the latest features, replace `0.13.0` with `1.0.0-SNAPSHOT` in the following guideline.
+
+### JDK
+
+Make sure to have the latest version of JDK 1.8 installed.
+
+### Android
+
+Arrow supports Android starting on API 21 and up.
+
+### Gradle
+
+#### Basic Setup
+
+In your project's root `build.gradle`, append these repositories to your list:
+
+```groovy
+allprojects {
+    repositories {
+        mavenCentral()
+        maven { url "https://oss.jfrog.org/artifactory/oss-snapshot-local/" } // for SNAPSHOT builds
+    }
+}
+```
+
+Add the dependencies into the project's `build.gradle`:
+
+##### Arrow Core
+
+```groovy
+apply plugin: 'kotlin-kapt'
+
+def arrow_version = "0.13.0"
+dependencies {
+    implementation "io.arrow-kt:arrow-core:$arrow_version"
+}
+```
+
+##### Arrow Core + Arrow Optics
+
+```groovy
+apply plugin: 'kotlin-kapt'
+
+def arrow_version = "0.13.0"
+dependencies {
+    implementation "io.arrow-kt:arrow-optics:$arrow_version"
```

Comment:
@aballano I'm not familiar with best gradle practices, but after reading the diff between API and impl makes sense. Probably we should make those changes in a different issue or PR since it involves build files and can potentially affect the way things are packaged for the release.